.github/workflows/ci_cmake.yml

```diff
@@ -0,0 +1,130 @@
+# Copyright (c) 2023-present The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+name: CI_CMAKE
+on: [push, pull_request]
+
```

Comment:
Why did you create a separate file instead of ci.yml?

Reply:
I wanted to keep the gnu build CI and cmake based ci in different files. The idea was that this one can be a basic smoke test. If this passes it can trigger the other tests which we'll configure with a matrix combination of dependencies, compilers and os.